app/routers/user_exercise.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Session
+from datetime import datetime
+from app.database.models import User, UserExercise
+from app.internal.utils import save
+
+
+def create_user_exercise(user: User, session: Session) -> UserExercise:
```

Comment:
On some function you put the user first, and on others the session is first. Stick to one convention

Reply:
fix the issue